### PR TITLE
create user not found exception

### PR DESCRIPTION
**Pull request by hussnainbutt568**
create successful responses

### CODE REVIEW

File: src/main/java/com/example/ecommercewebsite/Exceptions/ExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.example.ecommercewebsite.Exceptions;
+
+import com.example.ecommercewebsite.Response.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+@ControllerAdvice
+public class ExceptionHandler {
+    @org.springframework.web.bind.annotation.ExceptionHandler(UserNotFound.class)
```
Comment (JDevelopers568) [inline src/main/java/com/example/ecommercewebsite/Exceptions/ExceptionHandler.java:10]: Make it simple annotation is too long

Comment (hussnainbutt568) [reply src/main/java/com/example/ecommercewebsite/Exceptions/ExceptionHandler.java:10]: ok

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/ecommerce
 spring.datasource.username=root
-spring.datasource.password=fn03215169445
+spring.datasource.password=hussnain7212
```
Comment (JDevelopers568) [inline src/main/resources/application.properties:3]: please remove your password from application.properties before pushing

Comment (JDevelopers568) [inline src/main/resources/application.properties:3]: please remove password before pushing

**Review by JDevelopers568 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 2 above)
Comment (hussnainbutt568) [reply src/main/resources/application.properties:3]: just did

**Pull request closed without merging**